jwst/jump/tests/test_detect_jumps.py

```diff
@@ -506,7 +506,7 @@ def test_proc(setup_inputs):
     ngroups = 10
 
     model1, gdq, rnModel, pixdq, err, gain = \
-        setup_inputs( ngroups=ngroups, nrows=5, ncols=6, nints=2, gain=ingain, readnoise=inreadnoise,
+        setup_inputs( ngroups=ngroups, nrows=25, ncols=6, nints=2, gain=ingain, readnoise=inreadnoise,
```

Comment:
Reviewers: these couple of tests that employ multi-processing had their image sizes increased a little just to make sure there were enough rows in the image to sensibly divide up between the multiple processes.